### ISSUE TITLE
snow_height always zero

### ISSUE THREAD

**Issue by drmrbrewer** [open-meteo/open-meteo#51]
It seems that `snow_height` values are always zero, though related `freezinglevel_height` shows actual data... [example API call](https://api.open-meteo.com/v1/forecast?latitude=52.52&longitude=13.41&hourly=temperature_2m,precipitation,snow_height,freezinglevel_height)

**Comment by open-meteo** [comment]
There is currently no snow in Berlin ;-)

`freezinglevel_height` show the altitude above sea level at which point the air temperature drops below 0°C. This is around 2000 meters.

`snow_height` show the snow height / thickness on the ground.

I must admit the naming is not perfect, but those terms are commonly used. Was this the issue you tried to point out? Or is there another problem I overlooked?

**Comment by drmrbrewer** [comment]
Oh OK it's just a misunderstanding about what this parameter means.  Usually this would be referred to as "snow depth" in English, I think, not "snow height"... for example as per the [ICON docs](https://www.dwd.de/SharedDocs/downloads/DE/modelldokumentationen/nwv/icon/icon_dbbeschr_aktuell.pdf?view=nasPublication&nn=13934):

![image](https://user-images.githubusercontent.com/9009792/142028255-e8ed39b6-03ff-4d1c-8502-7b13612c6dc0.png)

and:

![image](https://user-images.githubusercontent.com/9009792/142028387-6a041b5f-ed24-43a4-8b17-b033a18d81bb.png)

and also [MOSMIX](https://opendata.dwd.de/weather/lib/MetElementDefinition.xml):

![image](https://user-images.githubusercontent.com/9009792/142028684-fe0ee994-10ac-4a6c-85b2-fd76d049217d.png)

I'd say that "snow height" suggests something more like the height at which precipitation will be falling as snow, like the following in ICON:

![image](https://user-images.githubusercontent.com/9009792/142028963-8658da09-0b24-4d1a-a88d-77cb0653fb98.png)

BTW is it possible to add the SNOWLMT (or equivalent) to open-meteo?  that's what I think of as "snow height" :-)

**Comment by open-meteo** [comment]
Good point. I will change it to `snow_depth` (but continue to support API calls with `snow_height` for interface stability).

`SNOWLMT` is only available in regional ICON models and not in global models. This would create a couple of edge cases in processing and API handling, that I did not anticipate for. I am also not sure if data will be that much different from freezing level height. Did you already use SNOWLMT from icon?

**Comment by drmrbrewer** [comment]
Fast work, thanks!

I see you've changed the parameter name and some of the phrases, but the following still remains:

![image](https://user-images.githubusercontent.com/9009792/142047197-11a321a7-26a8-4ea6-9a72-62bd71ab539e.png)

**Comment by open-meteo** [comment]
Good catch, I just changed it
